src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -70,12 +70,15 @@ sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLO
 ```
 
 For non-macOS devices building for iOS, you will need to pass extra parameters that can't be computed automatically (`--bundle-id` and `--version-name`).
+`--version-name` correspond to `CFBundleShortVersionString` within your `Info.plist`.
+
+ on iOS.
 
 ```bash
 sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLOYMENT} --dist {DIST} --bundle-id {BUNDLE_ID} --version-name {VERSION_NAME}
 ```
 
-If you don't want to use the automatically computed release format, you will need to pass the `--release-name` parameter.
+If you don't want to use the automatically computed release format, instead of passing `--bundle-id` and `--version-name` you will need to pass the `--release-name` parameter.
```

Comment:
```suggestion
If you don't want to use the automatically computed release format, instead of passing `--bundle-id` and `--version-name`, you'll need to pass the `--release-name` parameter.
```